.php_cs

```diff
@@ -13,6 +13,7 @@ $config = PhpCsFixer\Config::create()
             'syntax' => 'short'
         ],
         'blank_line_after_opening_tag' => true,
+        'blank_line_before_statement' => true,
```

Comment:
Note that this fixer can be configured to include more statements, for example:

```diff
diff --git a/.php_cs b/.php_cs
index c19236d..a9ea4d7 100644
--- a/.php_cs
+++ b/.php_cs
@@ -13,7 +13,30 @@ $config = PhpCsFixer\Config::create()
             'syntax' => 'short'
         ],
         'blank_line_after_opening_tag' => true,
-        'blank_line_before_statement' => true,
+        'blank_line_before_statement' => [
+            'statements' => [
+                'break',
+                'continue',
+                'declare',
+                'die',
+                'do',
+                'exit',
+                'for',
+                'foreach',
+                'goto',
+                'if',
+                'include',
+                'include_once',
+                'require',
+                'require_once',
+                'return',
+                'switch',
+                'throw',
+                'try',
+                'while',
+                'yield',
+            ],
+        ],
         'method_separation' => true,
         'no_alias_functions' => true,
         'no_empty_phpdoc' => true,
```

Generally, this has the advantage of giving the code a bit more vertical space, or in other words: room to breathe.